go.mod

```diff
@@ -18,13 +18,14 @@ require (
 	github.com/samber/lo v1.38.1
 	github.com/sirupsen/logrus v1.9.3
 	github.com/spf13/cobra v1.7.0
+	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.12.0
 	github.com/stretchr/testify v1.8.4
 	go.uber.org/goleak v1.2.1
 	golang.org/x/net v0.23.0
-	golang.org/x/sync v0.4.0
+	golang.org/x/sync v0.6.0
 	k8s.io/api v0.28.9
-	k8s.io/apimachinery v0.28.9
+	k8s.io/apimachinery v0.30.1
```

Comment:
We should keep the k8s libs within the same version to avoid any unexpected incompatibility issues. 🙏

Reply:
restored the version to the original one 🙏